js/interface.js

```diff
@@ -1313,6 +1323,10 @@ function renderSearchResult(result, searchType) {
 
   result = result
     .filter(function(item) {
+      if (deletedOnly && item.deletedAt) {
```

Comment:
Given you're returning `item.deletedAt` for assessing the filter, I don't think you need the `&& item.deletedAt` here.

```suggestion
      if (deletedOnly) {
```